packages/slate/src/models/range.js

```diff
@@ -302,8 +361,7 @@ class Range extends Record(DEFAULTS) {
    */
 
   hasAnchorAtEndOf(node) {
```

Comment:
I think we can deprecate these methods now. Previously they were required in "anchor" and "focus" were owned by the range, but now we can just have people call into the `Point` methods instead. So instead of:

```js
range.hasAnchorAtEndOf(node)
```

People can just as easily do:

```js
range.anchor.isAtEndOf(node)
```

And it keeps us from having to duplicate a lot of logic.